### PR TITLE
Support Laravel 6.0

### PR DESCRIPTION
**Pull request by DariusIII**
Add support for Laravel 6.0.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -12,9 +12,9 @@
     ],
     "require": {
         "php": ">=5.5.9",
-        "illuminate/support": "^5.1",
-        "illuminate/database": "^5.1",
-        "illuminate/events": "^5.1"
+        "illuminate/support": "^5.1|^6.0",
```
Comment (junaidnasir) [inline composer.json]: perhaps this should be changed to `^5.8|^6.0` if I am not mistaken.

**Review by junaidnasir — CHANGES_REQUESTED** [review]
thanks for pr. minor comment

File: composer.json (hunk 1 above)
Comment (DariusIII) [reply composer.json]: Actually, it should be 6.0 only, as 5.x does not support DeferrableProvider for service providers.

Edit: I am wrong, it does support it. Anyway ^5.1 also means 5.8 is supported

Comment (junaidnasir) [reply composer.json]: yes but the package shouldn't be installed for anyone having `<5.8` since it will break for them.

Comment (DariusIII) [reply composer.json]: I'll change it then.

**Comment by junaidnasir** [comment]
lgtm, unless you have any more changes I can merge it and release new version

**Comment by DariusIII** [comment]
It can be merged. Thank you.

**Pull request merged**